Extensions/TileMap/pixi-tilemap-helper.js

```diff
@@ -338,7 +338,7 @@
     pixiTileMap.clear();
 
     genericTileMapData.layers.forEach(function (layer, index) {
-      if (displayMode === 'index' && layerIndex !== index) return;
+      if (displayMode === 'index' && layerIndex != index) return;
```

Comment:
Let's cancel this change, we really must fix the problem upward. This is not the library issue if someones is not providing the proper types, it's the caller responsibility to follow the proper API.

Reply:
updated now :)